### DISCUSSION TITLE
14 Server and 3 Storage

### DISCUSSION THREAD

**Discussion by nk337** [swar/Swar-Chia-Plot-Manager#855] [Q&A]
Hi

I have a question. I run about 14 servers with 3 storage each 2 PiB. The 3 storages are run over 1 server as host and farmer. The other 14 servers are harvesters. 

The idea to run plotmanager on each harvester. 

Now my problem:

The storages are shared as network drive and have the path \\\storage\D. Can I specify this on the plot manager in the config.yaml in the job section:

Final_destination: \\\storage\d ?

That means can the Plot Manager access a network drive and store the final plots there ?

If this does not work I have to work with link expresion and set the folder from the network drive as junction on the desktop as a hardlink. 

Or are there other tools that can be started on a harvester ? The harvester have 8 TiB nvme ssd installed but no storage. The nvme ssd should only plot

**Comment by goliath525** [comment]
In my computer of each harvester, map a network drive with specific drive letter, then u can use the path in config.

**Reply by nk337** [reply]
Can you give me an example path ?
